test/gui/shared/scripts/bdd_hooks.py

```diff
@@ -48,6 +51,8 @@ def hook(context):
         context.userData['localBackendUrl'] = 'https://localhost:9200'
     if context.userData['clientSyncPath'] == '':
         context.userData['clientSyncPath'] = '/tmp/client-bdd/'
+    if context.userData['clientSyncPathUser'] == '':
+        context.userData['clientSyncPath'] = '/tmp/client-bdd-user/'
```

Comment:
may be you should rename the folders to `client-bdd-user1` and `client-bdd-user2` 🤔